roles/nsupdate_web/defaults/main.yml

```diff
@@ -5,8 +5,9 @@ nsupdate_web_port: "8080"
 nsupdate_web_ttl: "60"
 virtualenv_path: "~/venv"
 python_version: "python3"
-repo_url: "https://github.com/zmc/nsupdate-web.git"
-repo_path: "/home/{{ nsupdate_web_user }}/nsupdate_web"
+nsupdate_web_repo: "https://github.com/zmc/nsupdate-web.git"
```

Comment:
@zmc changed ownership (or copied) the nsupdate-web repo to https://github.com/ceph/nsupdate-web.  Can you change this line to reflect that please?

Reply:
@djgalloway I'm aware of it, but I would like to do this in separate PR

Reply:
Why not this PR?

Reply:
> Why not this PR?

The PR created before @zmc created the fork, and I've not tested it.

Reply:
I can submit a commit for this later here, but it will take some time before I test in my environment.

Reply:
Master branches are the same.  https://github.com/zmc/nsupdate-web/compare/master...ceph:master

I know I'm nit-picking but I'd rather not merge technical debt no matter how small.

Reply:
@djgalloway here you are

Reply:
Thank you!